dotnet/samples/KernelSyntaxExamples/Example48_GroundednessChecks.cs

```diff
@@ -146,7 +146,7 @@ public static async Task PlanningWithGroundednessAsync()
         Console.WriteLine($"======== Goal: ========\n{ask}");
         Console.WriteLine($"======== Plan ========\n{plan}");
 
-        var result = plan.Invoke(kernel, new KernelArguments(), CancellationToken.None);
+        var result = await plan.InvokeAsync(kernel, new KernelArguments(), CancellationToken.None);
```

Comment:
This doesn't look like changes you made but rather, changes that got in when you merged to main...

Something seems wrong that this appears in your PR...

Reply:
No, this is something I changed. The plan invoke function is now async.